app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,15 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str, favourite_dish: str,
+                 skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf blacksmith {self.nickname} with skill of "
+                + f"the {self._skill_level} level")
```

Comment:
The `player_info` method correctly constructs a string with the blacksmith's nickname and skill level. Ensure that the `nickname` attribute is accessible from the `Dwarf` class.